pkg/controller/infinispan/util/controllerutil.go

```diff
@@ -0,0 +1,138 @@
+package util
```

Comment:
As this is contained within the `util` package, I think we can simply call the file `pkg/controller/infinispan/util/controller.go`

Reply:
Fixed